### ISSUE TITLE
With API 35 androidx.test.uiautomator.UiObjectNotFoundException: UiSelector[RESOURCE_ID=loginId]

### ISSUE THREAD

**Issue by Aaron-Ritter** [FusionAuth/fusionauth-android-sdk#127]
### Description 
While running automated e2e testing on api level 35 the test is failing because it cant find the loginid element.

https://github.com/FusionAuth/fusionauth-android-sdk/actions/runs/12032155237/job/33544520792

```
io.fusionauth.sdk.FullEnd2EndTest > e2eTest[emulator-5554 - 15] FAILED 
	androidx.test.uiautomator.UiObjectNotFoundException: UiSelector[RESOURCE_ID=loginId]
	at androidx.test.uiautomator.UiObject.setText(UiObject.java:589)
Tests on emulator-5554 - 15 failed: There was 1 failure(s).
```

### Affects versions
api-level: 35 target: google_apis arch: x86_64

### Additional context
Problem is reported at https://issuetracker.google.com/issues/381518931.

**Comment by Aaron-Ritter** [comment]
workaround https://github.com/FusionAuth/fusionauth-android-sdk/pull/105/commits/1cea2b9669fece9bcb1b40f219210b1ece29e18f

**Comment by Aaron-Ritter** [comment]
Chrome version provided in VanillaIceCream, 35, x86_64, Android 15.0 (Google APIs) emulator image used in our tests is Chrome 124.0.6367.219.

When using VanillaIceCream, 35, x86_64, Android 15.0 (Google Play) the version of Chrome is as well 124.0.6367.219

We used the play image to easily update the chrome version to test if it is related to it. After updating to Chrome 131.0.6778.81 the tests works. 

But for automated testing we want to avoid having to update chrome every time we start a fresh emulator which is why with the workaround of using `textFields[0]` instead of `resourceId("loginId")` we implemented a solution which works on all standard images.

Great would be if the base image gets updated with the lastest chrome version bug i guess that's not possible.
